resources/templates/nodes.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
is there any risk that we, working with some particular DB replicaset, will override this file and commit/push by mistake?

it is kind of config, right? it will be individual for every project? where can we keep it (may we keep it outside the repo?)?